### PR TITLE
Adds Yattee to "Frontends"

### LINKED ISSUE #1954: Add Yattee - Privacy oriented video player for iOS, tvOS and macOS to "Frontends"
Videos browser and player for Invidious and Piped (alternative, privacy-friendly YouTube frontends) built for iOS, tvOS, and macOS. 

https://github.com/yattee/yattee

_Originally posted by @JmStyx in https://github.com/privacyguides/privacyguides.org/discussions/457_

## **Update - Since Rejection** 
The issue regarding not being able to support local subscriptions was overlooked during its original evaluation, as It's totally possible to use an instance Anonymously without being logged into an account. While holding a subscriptions lists or in Yattee's case it's called "Favorites".

## Features Include: 
- Native subscriptions without account through the use of Favorites 
- Native Proxy videos support
- Native user interface built with [SwiftUI](https://developer.apple.com/xcode/swiftui/) with customization settings
- Native Player queue and history (History is saved locally and can be cleared/disabled)
- Native Player component with custom controls, gestures and support for 4K playback
- Native Fullscreen, Picture in Picture and background audio playback
- Native [SponsorBlock](https://sponsor.ajay.app/), configurable categories to skip

### Extras:
Yattee supports an .json manifest file as the default app on the app store is bare-bones meaning you have to add your own custom locations or official manifest file as shown in my guide I have posted https://gonzoknows.com/posts/Yattee/
 - These manifest files can also be self-hosted and maintained to be shared

### PR DESCRIPTION
**Pull request by gonzoknows**
## Note:
I'm unable to upload __.svg__, so currently I've set the logo to the remote GitHub URL to pull the image from. This can be changed by someone who has higher permissions. 

### - closes #1954 and #457

- [x] Please check this box to confirm you have disclosed any relevant conflicts of interest in your post.
- [x] Please check this box to confirm your agreement to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute your contribution as part of our project.
- [x] Please check this box to confirm you are the sole author of this work, or that any additional authors will also reply to this PR on GitHub confirming their agreement to these terms.

### CODE REVIEW

**Comment by mfwmyfacewhen** [comment]
I downloaded Yattee to try it out and it's not loading any videos for me for some reason.

**Comment by gonzoknows** [comment]
> I downloaded Yattee to try it out and it's not loading any videos for me for some reason.

You need to add a piped or invidious instance to custom locations

**Comment by gonzoknows** [comment]
> I downloaded Yattee to try it out and it's not loading any videos for me for some reason.

Also, I would currently recommend using piped instances as invidious recently updated their API so Yattee still has to push an update as shown in this issue 
https://github.com/yattee/yattee/issues/391

**Comment by mfwmyfacewhen** [comment]
Thanks, I got it to work with your guide. Might link to that if that's ok with you, it's very helpful.

**Comment by gonzoknows** [comment]
> Thanks, I got it to work with your guide. Might link to that if that's ok with you, it's very helpful.

Definitely! I would appreciate that a lot, thanks.

File: docs/frontends.en.md [hunk 1]
```diff
@@ -114,6 +114,33 @@ When you are using a ProxiTok instance, make sure to read the privacy policy of
 !!! Warning
 
     When using FreeTube, your IP address may still be known to YouTube, [Invidious](https://instances.invidious.io) or [SponsorBlock](https://sponsor.ajay.app/) depending on your configuration. Consider using a [VPN](vpn.md) or [Tor](https://www.torproject.org) if your [threat model](basics/threat-modeling.md) requires hiding your IP address.
+    
+### Yattee
+
+!!! recommendation
+
+    ![Yattee logo](https://camo.githubusercontent.com/5acbbe712241861ba5c9fd0011e702f577b661f499f1fc7ffbcca32f37f4320c/68747470733a2f2f722e7961747465652e73747265616d2f69636f6e732f7961747465652d3135302e706e67){ align=right }
```
Comment (dngray) [inline docs/frontends.en.md:122]: We don't do third party images for the logo.

Is there a SVG image of this logo?

**Review by dngray — CHANGES_REQUESTED** [review]

File: docs/frontends.en.md (hunk 1 above)
Comment (mfwmyfacewhen) [reply docs/frontends.en.md:122]: Couldn't find one yet, that's why it's still a draft. Was meaning to ask the dev.

Comment (dngray) [reply docs/frontends.en.md:122]: I've emailed the main developer, maybe they will have one for us.

Comment (arekf) [reply docs/frontends.en.md:122]: Sorry, but I don't have this logo in SVG. What is more is that Apple has problem with it and was rejecting some App Store submissions with it so I will need to change it to something else.

I can let you know once I have a new logo in SVG in the future.

Comment (gonzoknows) [reply docs/frontends.en.md:122]: > Sorry, but I don't have this logo in SVG. What is more is that Apple has problem with it and was rejecting some App Store submissions with it so I will need to change it to something else.
> 
> I can let you know once I have a new logo in SVG in the future.

Thank you, once you have a .svg feel free to add it to this thread or email back a copy

Comment (dngray) [reply docs/frontends.en.md:122]: @arekf 

I recreated your logo as a SVG, was simple enough, looks pretty close to the original except maybe for the fillet radius on the outer object:

svg

Here's the SVG

yattee

**Review by dngray — APPROVED** [review]

**Review by jonaharagon — APPROVED** [review]

**Review by jonaharagon — CHANGES_REQUESTED** [review]
you reverted the fix

**Review by jonaharagon — APPROVED** [review]

**Comment by arekf** [comment]
Thanks to everyone for featuring Yattee on Privacy Guides ☺️ Especially to @dngray for taking effort to replicate the logo in SVG.

**Pull request merged**